scripts/deploy

```diff
@@ -96,6 +99,20 @@ else
   fi
 fi
 
+# check for `npm`
+if ! which npm; then
```

Comment:
Should we check for `node` here as well?

Reply:
npm requires node, and we require npm; this effectively checks for both.